README.md

```diff
@@ -112,7 +112,7 @@ To compare the output from [spritezero](https://github.com/mapbox/spritezero-cli
 ### Spritesheet size (total pixels)
 
 | Map style                | Spritezero pixels | Spreet pixels | Change |
-| :----------------------- | ----------------: | ------------: | -----: |
+|:-------------------------|------------------:|--------------:|-------:|
```

Comment:
I have Prettier set up to auto-format Markdown files on save, and the original style (with the spaces) comes from there. I'd like to leave this up to Prettier